multiqc/modules/damageprofiler/damageprofiler.py

```diff
@@ -227,7 +243,9 @@ def threeprime_plot(self):
         # Create tuples out of entries
         for key in self.threepGtoAfreq_data:
             pos = list(range(1,len(self.threepGtoAfreq_data.get(key))))
-            tuples = list(zip(pos,self.threepGtoAfreq_data.get(key)))
+            #Multiply values by 100 to get %
+            tmp = [i * 100 for i in self.threepGtoAfreq_data.get(key)]
```

Comment:
You can easily get caught out with variable types here if you're not careful. I basically always multiply by `100.0` so that you get floats with proper accuracy.

Reply:
Added it :-)